src/App.js

```diff
@@ -1,28 +1,60 @@
-import logo from './logo.svg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import { useEffect, useState } from 'react';
+import Footer from './components/common/Footer';
+import Header from './components/common/Header';
+import Flex from './components/common/Flex';
+import Slider from './components/common/Slider';
+import ImageContainer from './components/common/Image';
+import Card from './components/common/Card';
+import { results as productCategoriesResults } from './utils/product-categories.json';
+import { results as featuredBannersResults } from './utils/featured-banners.json';
+import { results as featuredProductsResults } from './utils/featured-products.json';
 
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
-
+  const [productsCategories, setProductsCategories] = useState([]);
+  const [featuredBanners, setFeaturedBanners] = useState([]);
+  const [featuredProducts, setFeaturedProducts] = useState([]);
+  
+  useEffect(() => {
+    setProductsCategories(productCategoriesResults);
+    setFeaturedBanners(featuredBannersResults);
+    setFeaturedProducts(featuredProductsResults);
+  },[])
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Header/>
+      <Flex>
+        <Slider items={featuredBanners} />
+      </Flex>
+      <hr/>
+      <Flex wrap={"wrap"} justifyContent={"space-around"} alignItems={"flex-start"}>
+        {productsCategories.map((item) => (
+          <Card>
+            <ImageContainer width={100} key={item.id} src={item.data.main_image.url} alt={item.data.alt}/>
+          </Card>
+        ))}
+      </Flex>
+      <hr/>
+      <Flex wrap={"wrap"} justifyContent={"space-around"} alignItems={"flex-start"}>
+        {featuredProducts.map((item) => (
+          <Card>
+            <Flex direction={"column"} alignItems={"centerolv"}>
+              <ImageContainer width={35} key={item.id} src={item.data.mainimage.url} alt={item.data.alt}/>
+              <div>
+                <Flex direction={"row"} justifyContent={"space-around"}>
+                  <p>Product name: {item.data.name}</p>
+                  <p>Price: ${item.data.price}</p>
+                  <p>Category:{item.data.category.slug}</p>
+                </Flex>
+              </div>
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏As your app grows, you may consider split this into smaller components, and separate concerns as you may need handlers and more functions on each sections, such products or banner, that even you can reuse in a future.